server/main.lua

```diff
@@ -253,7 +253,7 @@ Citizen.CreateThread(function()
                 NADMIN.refreshNameCache()
                 NADMIN.refreshBanCache()
                 
-                NADMIN.LogToAdmins(("Staff ^1%s^7 desbanea a ^1%s^7 (%s)"):format(xPlayer.getName(),(bannedidentifier~="NAC" and namecache[bannedidentifier]) and namecache[bannedidentifier] or "Desconocido",bannedidentifier))
+                NADMIN.LogToAdmins(("Staff ^1%s^7 unbalance ^1%s^7 (%s)"):format(xPlayer.getName(),(bannedidentifier~="NAC" and namecache[bannedidentifier]) and namecache[bannedidentifier] or "A stranger",bannedidentifier))
```

Comment:
The translations may be 'unbanned by' for the literal translation: '%s has been unbanned by %s (%s)'
```suggestion
                NADMIN.LogToAdmins(("^1%s^7 has been unbanned by ^1%s^7 (%s)"):format(bannedidentifier, (bannedidentifier~="NAC" and namecache[bannedidentifier]) and namecache[bannedidentifier] or "A stranger"))
```